nixos/modules/programs/vscode.nix

```diff
@@ -36,6 +36,18 @@ in
       description = "List of extensions to install.";
     };
 
+    policies = lib.mkOption {
+      type = lib.types.attrs;
+      default = { };
+      example = lib.literalExpression ''
+        {
+          "UpdateMode" = "none";
+          "TelemetryLevel" = "off";
+        }
+      '';
+      description = "System-wide policies for VS Code in /etc/vscode/policy.json";
```

Comment:
I'd suggest to include a link to the [upstream documentation](https://code.visualstudio.com/docs/setup/enterprise) of this feature.